lib/interface_quda.cpp

```diff
@@ -297,6 +302,13 @@ static void profilerStop(const char *f) {
 
 namespace quda {
   void printLaunchTimer();
+
+  void massRescale(cvector_ref<ColorSpinorField> &b, QudaInvertParam &param, bool for_multishift);
```

Comment:
Is there an `extern` or something of the sort missing on this declaration? I see how these routines are defined in the new `lib/solve.cpp` file... but I don't see a corresponding header definition?

Reply:
All function declarations are implicitly `extern` in C++.

Reply:
Today I learned!